src/screens/OtherProfile/index.js

```diff
@@ -391,13 +484,13 @@ const OtherProfile = () => {
     try {
       const processPostApi = await unblockUserApi({userId: dataMain.user_id});
       if (processPostApi.code === 200) {
-        refetchBlockStatus();
+        checkUserBlockHandle(dataMain.user_id);
         blockUserRef.current.close();
         specificIssueRef.current.close();
         reportUserRef.current.close();
       }
     } catch (e) {
-      refetchBlockStatus();
+      checkUserBlockHandle(dataMain.user_id);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `checkUserBlockHandle` function is called after unblocking a user. Ensure that this function correctly updates the state and UI to reflect the change in block status.
```diff
+ checkUserBlockHandle(dataMain.user_id);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->